README.md

```diff
@@ -29,15 +29,26 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+#### Dependencies
 
-    $ sudo gem install jekyll bundler
+1. Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS.
+2. Using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
+3. Install jekyll bundler by typing the following command:
```

Comment:
I think(no idea of ruby) jekyll and bundler are 2 gems. In that case add an `and` between the two and preferably bold them.

Reply:
@frextrite plz see https://jekyllrb.com/docs/. Here the command `gem install jekyll bundler` is given

Reply:
He is trying to say, that where you have writtem `Install jekyll nundler` there you should write `Install jekyll and bundler gems`

Reply:
and try to ommit `by typing the following command`. It is self-explanatory to user that he has to run this command in the terminal

Reply:
ohh ok, got it :sweat_smile:

Reply:
According to the site Jekyll and Bundler are gems, so you can edit this line to just `Install Jekyll by 
 typing ...`

Reply:
I agree with @KVGarg. Omitting `by typing the following command` would be better.